util/get-supported-module-env.js

```diff
@@ -4,95 +4,82 @@ const path = require('path');
 // See https://stackoverflow.com/a/75281896/10936596
 const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
 
-const URL = 'https://raw.githubusercontent.com/silverstripe/supported-modules/gh-pages/modules.json';
+/**
+ * Gets information for any "supported-module" repositories.
+ */
+async function getRelevantModules(jsonUrl, foundModules) {
+  const response = await fetch(jsonUrl);
+  const currentModules = await response.json();
+  // Only list supported-module, since the CMS 4 list includes some things we don't control or care about with other types.
+  const relevant = currentModules.filter(module => module.type === 'supported-module');
+  for (const module of relevant) {
+    foundModules[module.github] = module;
+  }
+}
```

Comment:
```suggestion
async function getRelevantModules(jsonUrl, foundModules) {
  await fetch(jsonUrl)
    .then(response => response.json())
    .then(currentModules => {  
      // Only list supported-module, since the CMS 4 list includes some things we don't control or care about with other types.
      const relevant = currentModules.filter(module => module.type === 'supported-module');
      for (let module of relevant) {
        foundModules[module.github] = module;
      }
    });
}
```

Slightly more standard use of fetch()

Reply:
That's not a more standard way to use asnyc functions, and doesn't follow [the documentation for node-fetch](https://www.npmjs.com/package/node-fetch#json).

Reply:
> That's not a more standard way to use asnyc functions

I'd say it's a more standard way to use fetch() specifically, means you can do the the `if (!response.ok) { throw new Error() } ... .catch(error => { ... }` pattern, though since we're not trying to handle things going wrong here we probably don't need to use `.then()`